streaming/index.js

```diff
@@ -182,14 +181,69 @@ const CHANNEL_NAMES = [
 ];
 
 const startServer = async () => {
+  const pgPool = new pg.Pool(pgConfigFromEnv(process.env));
+  const server = http.createServer();
+  const wss = new WebSocket.Server({ noServer: true });
```

Comment:
The reason for not passing the `server` option of the http Server above is because we want to use a custom upgrade handler.